db/models.py

```diff
@@ -49,4 +57,52 @@ class MovieSession(models.Model):
     )
 
     def __str__(self) -> str:
-        return f"{self.movie.title} {str(self.show_time)}"
+        return f"{self.movie.title} {self.show_time}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to=get_user_model(),
+                             on_delete=models.CASCADE,
+                             related_name="orders")
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+    class Meta:
+        ordering = ("-created_at",)
```

Comment:
The `class Meta` should be defined after all attributes in the `Order` class. According to the checklist, implement `class Meta` only after you defined all attributes.